packages/assets-controllers/src/TokenListController.ts

```diff
@@ -113,36 +111,26 @@ export class TokenListController extends StaticIntervalPollingController<TokenLi
 
   private readonly cacheRefreshThreshold: number;
 
-  private chainId: Hex;
```

Comment:
We are going to need this. During fetchFromTokenList, we want to correctly update `tokenList` if the selectedNetwork matches the current chainId.

When polling across chains, we want to update `tokensChainsCache` for each chain.
When currentChainId === chainId, also update `tokenList` for backwards compatability